webpack.config.js

```diff
@@ -1,11 +1,13 @@
-var path = require("path");
-var webpack = require("webpack");
-var HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+const webpack = require("webpack");
+const CleanWebpackPlugin = require("clean-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
-var BUILD_DIR = path.resolve(__dirname, "dist");
-var APP_DIR = path.resolve(__dirname, "src");
+const BUILD_DIR = path.resolve(__dirname, "dist");
+const APP_DIR = path.resolve(__dirname, "src");
 
-var config = {
+const config = {
     entry: ["es6-shim", APP_DIR + "/index.tsx"],
```

Comment:
replace es6-shim by babel polyfill (check command center project)
It's a fix about ie 11 / edge bugs